packages/create-freesewing-pattern/template/freesewing/example/src/App.js

```diff
@@ -26,17 +25,19 @@ const App = (props) => {
   // Adds support for loading an external pattern configuration
   let recreate = false
   if (window) recreate = window.location.pathname.substr(1).split('/')
-  if (recreate.length === 3 && recreate[0] === 'recreate')
+  if (recreate.length === 3 && recreate[0] === 'recreate') {
     recreate = { from: recreate[1], id: recreate[2] }
-  else recreate = false
+  } else {
+    recreate = false
+  }
 
   return (
     <Workbench
       freesewing={freesewing}
       Pattern={Pattern}
-      userLanguage="{{language}}"
+      userLanguage="en"
```

Comment:
Just noticing this might be inadvertently changed. Needs to be resolved before merging.

Reply:
Put this back to be the placeholder